aictl/cli/main.py

```diff
@@ -403,6 +449,46 @@ def main():
     )
     t2a_parser.set_defaults(func=t2a)
 
+    ####Upscale
+    upscale_parser = subparsers.add_parser("upscale", help="Upscale an image")
+    upscale_parser.add_argument("-p", "--prompt", default="", help="the prompt to use")
```

Comment:
In the help section of this argument, let's make it explicit that this argument is only available for the stable diffusion model.

Reply:
done